forest/shared/src/cli/config.rs

```diff
@@ -217,6 +141,7 @@ mod test {
 
     /// Partial configuration, as some parts of the proper one don't implement
     /// required traits (i.e. Debug)
+    // TODO(aatifsyed): the above comment is wrong
```

Comment:
If it is wrong now (as the documentation tends to lag behind the code changes) then please modify it to reflect the current state. A wild TODO is, in general, not very helpful.

Reply:
Or, even better, let's remove this structure altogether. It can be done in a different PR, but let's not leave untracked TODOs behind.

Reply:
#2965